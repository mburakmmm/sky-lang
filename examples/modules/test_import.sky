# Import test

import math_utils

function main
  print("=== Import Test ===")
  
  let sum = add(5, 3)
  print("5 + 3 =")
  print(sum)
  
  let product = multiply(4, 7)
  print("4 * 7 =")
  print(product)
  
  let sq = square(6)
  print("6^2 =")
  print(sq)
  
  print("=== Done ===")
end

