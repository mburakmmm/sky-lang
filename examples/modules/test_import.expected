=== Import Test ===
5 + 3 =
8
4 * 7 =
28
6^2 =
36
=== Done ===

